deployment/operator/config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ghcr.io/containers/nri-plugins-operator
+  newName: ghcr.io/containers/nri-plugins/nri-plugins-operator
```

Comment:
While we're fixing this... I'd prefer to have a `yaml.in` file (which is part of the repository), which is then copied and [s]edited into the resulting `.yaml` (which is not part of the repository). Could you fix this as well ?

Otherwise in the current state `make images`  (`make  nri-plugins-operator-bundle-image` more specifically) always leaves the repository with changed files in an uncommitted state which one needs to undo with a `git checkout deployment/operator/config/manager/kustomization.yaml` or something else to the same effect.

Reply:
@fmuyassarov For instance something like this: #224